catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -82,8 +73,9 @@ private void setSubject(Operation operation) {
             if (!(requestSubject instanceof ddf.security.Subject)) {
                 subject = SecurityUtils.getSubject();
```

Comment:
Could declare subject here and avoid the one silly warning reported for this file.